src/error.js

```diff
@@ -355,3 +367,43 @@ export function detectNonAmpJs(win) {
 export function resetAccumulatedErrorMessagesForTesting() {
   accumulatedErrorMessages = [];
 }
+
+export function detectJsEngineFromStack() {
+  const object = Object.create({
+    // DO NOT rename this property.
+    // DO NOT transform into shorthand method syntax.
+    t: function() {
+      throw new Error('message');
+    }
+  });
+  try {
```

Comment:
K. Please update errortracker.go to append this to the UA string (maybe wrapped in [] or something like that).